lib/web/integrations_awsra_test.go

```diff
@@ -124,3 +127,55 @@ func TestBuildAWSRATrustAnchorConfigureScript(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateAWSRolesAnywhereIntegration(t *testing.T) {
+	t.Parallel()
+	wPack := newWebPack(t, 1 /* proxies */)
+	proxy := wPack.proxies[0]
+	authPack := proxy.authPack(t, "user", []types.Role{services.NewPresetEditorRole()})
+	ctx := t.Context()
+
+	existingIntegration, err := types.NewIntegrationAWSOIDC(types.Metadata{
+		Name: "existing-integration",
+	}, &types.AWSOIDCIntegrationSpecV1{
+		RoleARN: "arn:aws:iam::123456789012:role/valid-role",
+	})
+	require.NoError(t, err)
+
+	_, err = wPack.server.Auth().CreateIntegration(ctx, existingIntegration)
+	require.NoError(t, err)
+
+	for _, tt := range []struct {
+		name            string
+		integrationName string
+		errCheck        require.ErrorAssertionFunc
+		errContains     string
```

Comment:
no need to have a separate erroContains field, you can have error contains check within the `ErrorAssertionFunc`?